source/iostreams.tex

```diff
@@ -15883,10 +15882,7 @@
 
 \pnum
 \begin{note}
-For purposes of exposition,
-class \tcode{directory_iterator}\iref{fs.class.directory.iterator}
-is shown above as a friend of class \tcode{directory_entry}.
-Friendship allows the \tcode{directory_iterator} implementation to cache
+\tcode{directory_iterator} is allowed to cache
```

Comment:
Are we allowed to say "allowed"?
Should this be turned into a "Recommended practice" note and say something like "Implementations of `directory_iterator` are encouraged to populate attributes of the `directory_entry` object upon construction, without calling `refresh()`"?
OTOH does this note actually mean anything at all? Implementations are _always_ encouraged to omit "unneeded" code; we don't have to tell them that, do we?

Reply:
Changed to use "can". (So did #6758.)